package.json

```diff
@@ -26,26 +26,37 @@
     "@babel/preset-typescript": "7.10.4",
     "@percy/storybook": "3.3.0",
     "@storybook/addon-docs": "6.0.0-rc.30",
+    "@storybook/addon-essentials": "6.0.6",
     "@storybook/addon-knobs": "6.0.0-rc.30",
-    "@storybook/preset-create-react-app": "3.1.4",
     "@storybook/react": "6.0.0-rc.30",
-    "@typescript-eslint/eslint-plugin": "3.9.0",
-    "@typescript-eslint/parser": "3.9.0",
+    "@typescript-eslint/eslint-plugin": "2.x",
```

Comment:
The dependencies should probably be pinned.

Reply:
Done, thanks.